src/Service.Host/.babelrc

```diff
@@ -13,8 +13,8 @@
     [
       "@babel/plugin-proposal-class-properties",
       {
-        "loose": true
+        "loose": false
       }
```

Comment:
you can just omit this entirely since false is the default . Doesn't really matter though